.travis/BuildLinux.sh

```diff
@@ -61,18 +49,20 @@ if [ -z "${RUN_CLANG_TIDY}" ] \
         time make test-libs-domain -j2
         time make test-libs-elliptic -j2
         time make test-libs-evolution -j2
-        time make test-libs-numerical-algorithms -j2
     fi
 
     if [[ ${TRAVIS_BUILD_STAGE_NAME} = "Build test libraries" ]]; then
-        time make test-libs-data-structures -j2
+        time make test-libs-numerical-algorithms -j2
+        # Build DataStructures in serial because the DataVector tests
+        # are very memory intensive
```

Comment:
I suggest clarifying: "very memory intensive to compile"

Reply:
I can't (or at least super not easily) profile the usage on Travis so I can't give numbers, but I'm happy to add more to the comment if you give me some guidelines as to what to say :)

Reply:
Oh, sorry I wasn't super clear, I meant clarifying "memory intensive" -> "memory intensive to compile" in your comment, so as to avoid any ambiguity about compile-time vs run-time memory use. :)

Reply:
Ah, good point! Will do :)